### DISCUSSION TITLE
Clerk Example

### DISCUSSION THREAD

**Discussion by verdverm** [vikejs/vike#1503] [Discussion]
I'm working on integrating Clerk with Vike. I thought I'd reach out about contributing an example, and will probably need some guidance along the way. Is there anything I should know beyond the `Contributing` doc?

I'm starting from the `examples/auth`, but it looks like a Clerk integration will need to change it a bit

My first questions are:

1. should I put the example under `examples/auth-clerk`?
2. in terms of "minimal", I'd like to show webhooks, possibly updating a database record? Is the database too much for a minimal showing of the integration?
3. Is it ok to use tailwindcss and a different layout? Or do you prefer the consistency between the examples layout / content?

**Comment by brillout** [comment]
Sounds great! How about we host the example under your username `github.com/verdverm` and add link(s) to it in the docs? That way you'll have the freedom to showcase other things as well.
